docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: "3.8"
+services:
+  db:
+    image: postgres:14.3-alpine
+    volumes:
+      - ./data/db:/var/lib/postgresql/data/
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_DB=postgres
+
+  django:
+    restart: always
+    container_name: backend
```

Comment:
Container name for all 3 should be more descriptive (ex: postgres, django, react)